File: Python/apaxiaaans.py
'''
 *
 *	Problem:		Apaxiaaaaaaaaaaaans!
 *	Problem ID:	apaxiaaans
 *	url:				https://open.kattis.com/problems/apaxiaaans
 *	Solution:		Python
 *
'''

name = input()
output = name[0]

for i in (range(1,len(name))):
	if not(name[i] == name[i-1]):
		output += name[i]
	
print(output)

File: Ruby/apaxiaaans.rb
=begin
 *
 *	Problem:		Apaxiaaaaaaaaaaaans!
 *	Problem ID:	apaxiaaans
 *	url:				https://open.kattis.com/problems/apaxiaaans
 *	Solution:		Ruby
 *
=end

class Solver

	def solve
		name = gets.chomp.split("")
		
		output = name.fetch(0)
		for index in 1...name.size
			prevChar = name.fetch(index - 1)
			char = name.fetch(index,1)
			unless (char == prevChar)
				output += char
			end
		end
		
		puts output
	end # solve method
	
end # end Main class

solver = Solver.new
solver.solve
